### ISSUE TITLE
Crashes on server launch 1.19

### ISSUE THREAD

**Issue by Cm4nXD** [audaki/minecraft-cart-engine#13]
[23:26:48 ERROR] [main]: [FabricLoader/Mixin] Mixin apply for mod audaki_cart_engine failed audaki_cart_engine.mixins.json:AbstractMinecartEntityMixin from mod audaki_cart_engine -> net.minecraft.class_1688: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow method getMaxOffRailSpeed in audaki_cart_engine.mixins.json:AbstractMinecartEntityMixin from mod audaki_cart_engine was not located in the target class net.minecraft.class_1688. Using refmap audaki-cart-engine-refmap.json

**Comment by audaki** [comment]
@Cm4nXD Hey, thanks for the issue. Which version is the server exactly, is it 1.19.0 or .1 or .2?

**Comment by pajicadvance** [comment]
I have the same problem on a 1.19.2 client.

**Comment by audaki** [comment]
@Cm4nXD @pajicadvance Sorry for the inconvenience, I learned something today. Loom/Yarn sometimes changes method names (probably to better reflect the real source code) and this happened to Cart code. The method was renamed from getMaxOffRailSpeed to getMaxSpeed. Maybe Mojang changed the method name?

Anyway, I will look for this in the future and I've released a new version 1.1.0 and I tested it on Windows with Minecraft 1.19.2 and the newest Fabric API and Fabric Loader and it worked fine:

https://www.curseforge.com/minecraft/mc-mods/audaki-cart-engine/files
